public/static-checkout/index.html

```diff
@@ -0,0 +1,148 @@
+        <a class="dropdown-item" href="https://courses.edx.org/logout">Logout</a></div>
+    </div>
+  </header>
+  <main>
+    <div class="container-fluid payment-container" aria-label="Payment">
+      <div class="row">
+        <!-- Cart  -->
+        <div class="cart col-5">
+          <!-- Product Line Item -->
+          <section aria-live="polite" aria-relevant="all" aria-label="Shopping Cart Details">
+            <div class="checkout-section" aria-label="Shopping cart details are loaded.">
+              <h5>In Your Cart</h5>
+              <p>Your purchase contains the following:<p>
+              <div class="row">
+                <div style="display: inline-flex;">
+                  <img class="img-thumbnail" src="https://prod-discovery.edx-cdn.org/media/course/image/ac499cd6-a3ac-4a5f-a307-bb28ea318de1-0a473d1ec055.small.png" alt="" />
+                  <div class="product-line-item">
+                    <h4>Title HERE</h4>
+                    <p>Verified Certificate</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </section>
+          <!-- Summary -->
+          <div class="checkout-section" role="region" aria-labelledby="summary-heading" aria-live="polite">
+            <h5>Summary</h5>
+            <p class="col-10">Price</p>
+            <span>$350.00</span>
+            <!-- Coupon form  -->
+            <p>Add coupon code (optional)</p>
+            <form style="display: inline-flex; margin-bottom: 1rem;">
+              <input style="margin-right: 0.5rem; width: 12rem;" type="text" name="couponcode" aria-describedby="">
+              <button class="submit-button" style="font-size: 1rem; width: 6.5rem; font-weight: 400;">Apply</button>
+            </form>
+            <h4 class="col-10">TOTAL</h4>
+            <span><strong>$350.00</strong></span>
+          </div>
+          <!-- Order Details -->
+          <div class="checkout-section">
+            <h5>Order Details</h5>
+            <p>After you complete your order you will be automatically enrolled in the verified track of the course.</p>
+          </div>
+        </div>
+
+        <!-- Checkout  -->
+        <div class="checkout col-7">
+          <section aria-label="Payment Details">
+            <!-- Payment Method  -->
+            <div class="checkout-section">
+              <h5>Select Payment Method</h5>
+              <div class="payment-method">
+                <button class="active-button"><img src="assets/accepted-card-logos.png" alt="card-logos"></button>
+                <button><img src="assets/paypal-logo.png" alt="paypal-logo"></button>
+              </div>
+            </div>
+            <!-- Checout Form  -->
+            <div class="card">
+              <div class="checkout-section">
+                <h5>Card Holder Infomation</h5>
+                  <form>
+                    <div class="col-left">
```

Comment:
perhaps these classes could be changed to cl and cr since they reoccur to save space?

Reply:
I've changed the CSS to use the .col classes I had created instead of floating.